File: fflocation/fflServer/src/model/UserInfo.java
package model;

/**
 * This class represents a user's information with it's own
 * id, nick, name, surname, email, phone, country, address and whether it is or not an administrator.
 * It has also a field to hold the user's last position.
 */
public class UserInfo {
	private int id;
	private String nick;
	private String name;
	private String surname;
	private String email;
	private int phone;
	private String country;
	private String address;
	private boolean isAdministrator;
	private Position lastPosition;
	
	public String toString(){
		return id + "/" + nick + ":" +
			"[" + name + " " + surname + "]" +
			"{" + email + " " + phone + "}" + 
			"(" + country + " " + address + ") " +
			"admin=" + isAdministrator + " ¿" + lastPosition + "?";
	}
	
	public UserInfo()
	{
		this.id = 0;
		this.nick = null;
		this.name = null;
		this.surname = null;
		this.email = null;
		this.phone = 0;
		this.country = null;
		this.address = null;
		this.isAdministrator = false;
		this.lastPosition=null;
	}
	
	public UserInfo(int id, String nick, 
			String name, String surname,
			String email, int phone, String country, String address,
			boolean isAdministrator)
	{
		
		this.id = id;
		this.nick = nick;
		this.name = name;
		this.surname = surname;
		this.email = email;
		this.phone = phone;
		this.country = country;
		this.address = address;
		this.isAdministrator = isAdministrator;
	}
	
	public void setNick(String nick) {
		this.nick = nick;
	}
	public String getNick() {
		return nick;
	}
	public void setId(int id) {
		this.id = id;
	}
	public int getId() {
		return id;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	public String getName() {
		return name;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getEmail() {
		return email;
	}
	public void setSurname(String surname) {
		this.surname = surname;
	}
	public String getSurname() {
		return surname;
	}
	public void setPhone(int phone) {
		this.phone = phone;
	}
	public int getPhone() {
		return phone;
	}
	public void setCountry(String country) {
		this.country = country;
	}
	public String getCountry() {
		return country;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getAddress() {
		return address;
	}
	public void setAdministrator(boolean isAdministrator) {
		this.isAdministrator = isAdministrator;
	}
	public boolean isAdministrator() {
		return isAdministrator;
	}
	public Position getPosition() {
		return lastPosition;
	}
	public void setPosition(Position p) {
		lastPosition=p;
	}

}

File: fflocation/fflwebsite/web_services/userinfo.php
<?
      /**
	This class is needed so as to communicate 
	with the server side. Instead of the two sides
	sending strings they send Objects type "UserInfo"
	for getting the user's information.
      */
      class UserInfo {
	    private $id;
	    private $nick;
	    private $name;
	    private $surname;
	    private $email;
	    private $phone;
	    private $country;
	    private $address;
	    private $administrator;
	    private $position;
		
	    public function toStringUserInfo(){
		  return $this->id . "/" . $this->nick . ":" .
			"[" . $this->name . " " . $this->surname . "]" .
			"{" . $this->email . " " . $this->phone . "}" . 
			"(" . $this->country . " " . $this->address . ") " .
			"admin=" . $this->administrator . " ¿" . $this->position . "?";
	    }
		
	    public function __construct() {
		  $this->id = 0;
		  $this->nick = null;
		  $this->name = null;
		  $this->surname = null;
		  $this->email = null;
		  $this->phone = 0;
		  $this->country = null;
		  $this->address = null;
		  $this->administrator = false;
		  $this->position=null;
	    }
		
	    public function setNick($nick1) {
		  $this->nick=$nick1;
	    }
		
	    public function getNick() {
		  return $this->nick;
	    }
		
	    public function setId($id1) {
		  $this->id=$id1;
	    }
		
	    public function getId() {
		  return $this->id;
	    }
		
	    public function setName($name1) {
		  $this->name=$name1;
	    }
		
	    public function getName() {
		  return $this->name;
	    }
		
	    public function setEmail($email1) {
		  $this->email=$email1;
	    }
		
	    public function getEmail() {
		  return $this->email;
	    }
		
	    public function setSurname($surname1) {
		  $this->surname=$surname1;
	    }
		
	    public function getSurname() {
		  return $this->surname;
	    }
	    
	    public function setPhone($phone1) {
		  $this->phone=$phone1;
	    }
		
	    public function getPhone() {
		  return $this->phone;
	    }
		
	    public function setCountry($country1) {
		  $this->country=$country1;
	    }
		
	    public function getCountry() {
		  return $this->country;
	    }
		
	    public function setAddress($address1) {
		  $this->address=$address1;
	    }
		
	    public function getAddress() {
		  return $this->address;
	    }
		
	    public function setAdministrator($isAdministrator1) {
		  $this->administrator=$isAdministrator1;
	    }
		
	    public function isAdministrator() {
			
		  return $this->administrator;
	    }
		
	    public function getPosition() {
		  return $this->position;
	    }
		
	    public function setPosition($p1) {
		  $this->position=$p1;
	    }
      }
?>
